gcovr/tests/html-css/style.css

```diff
@@ -0,0 +1,344 @@
+:root { font-family: sans-serif; }
```

Comment:
Test can be minimized by replacing custom CSS with a placeholder like `/* custom CSS */` – anything as long as it's clearly distinct from the default CSS. If we decide to turn custom CSS into Jinja templates we'd also need to include an example variable somewhere.

Reply:
Ok.